packages/foreman/rubygem-font-awesome-sass/rubygem-font-awesome-sass.spec

```
@@ -32,7 +32,7 @@ Documentation for %{name}.
 
 %prep
 %setup -q -n  %{gem_name}-%{version}
-%patch0 -p1
+%patch -P 0 -p1
```

Should this use [autosetup](https://rpm-software-management.github.io/rpm/manual/autosetup.html) instead?
```suggestion
%autosetup -p1-n %{gem_name}-%{version}
```

---

That seems to work as well